modules/nextflow/src/main/groovy/nextflow/cli/CmdConfig.groovy

```diff
@@ -123,6 +135,21 @@ class CmdConfig extends CmdBase {
         output << ConfigHelper.toPropertiesString(config, sort)
     }
 
+    /**
+     * Prints a property of a {@link ConfigObject}.
+     *
+     * @param config The {@link ConfigObject} representing the parsed workflow configuration
+     * @param name The {@link String} representing the property name using dot notation
+     * @param output The stream where output the formatted configuration notation
+     */
+    @PackageScope void printProperty(ConfigObject config, String name, OutputStream output) {
+        final map = config.flatten()
+        if( !map.containsKey(name) )
+            throw new IllegalArgumentException("Property '$name' not found")
```

Comment:
```suggestion
            throw new AbortOperationException("Configuration property '$name' not found")
```

Using `AbortOperationException` should provide a slightly better error message

Reply:
My bad, @afishman you had it right the first time 😄